website/source/docs/platform/k8s/connect.html.md

```diff
@@ -174,10 +174,28 @@ Annotations can be used to configure the injection behavior.
 
 * `consul.hashicorp.com/connect-service-upstreams` - The list of upstream
   services that this pod needs to connect to via Connect along with a static
-  local port to listen for those connections. Example: `db:1234,auth:6789`
-  will start two local listeners for `db` on port 1234 and `auth` on port
-  6789, respectively. The name of the service is the name of the service
-  registered with Consul. This value defaults to no upstreams.
+  local port to listen for those connections. 
+
+    * Services
+   
+        The name of the service is the name of the service registered with Consul. Datacenters can optionally be specified within this annotation. 
+        ```annotations:
+        "consul.hashicorp.com/connect-service-upstreams":"[service-name]:[port]:[optional datacenter]"
+        ```
+
+    * Multiple Services
+
+        If multiple services would like to be specified they are delimited by commas.
```

Comment:
Active voice too: If you would like to specify multiple services, delimit them with commas.

Reply:
Thank you, should be resolved :)